lib/src/main/java/graphql/nadel/Nadel.kt

```diff
@@ -237,24 +231,49 @@ class Nadel private constructor(
         private var schemaTransformationHook: SchemaTransformationHook = SchemaTransformationHook.Identity
         private var engineFactory: NadelExecutionEngineFactory = NadelExecutionEngineFactory(::NextgenEngine)
 
-        fun dsl(serviceName: String, nsdl: Reader): Builder {
-            Objects.requireNonNull(nsdl)
-            serviceNDSLs[serviceName] = nsdl
+        private var schemaBuilder = NadelSchemas.Builder()
```

Comment:
Could change this to supply the built `NadelSchemas` object in the future, but I don't think that's necessary right now.